.gitignore

```diff
@@ -1,18 +1,16 @@
-*.log
-*.pot
-*.pyc
-__pycache__
-.vagrant
-build
-ENV
-.coverage
-htmlcov
+# Consider using a global git ignore file for files and directories that
+# are commonly generated by your OS, IDE, Python, development tools etc.
+# A fairly extensive starting point would be:
+#
+# https://www.gitignore.io/api/windows,linux,osx,python,vagrant,sass,idea,visualstudiocode
```

Comment:
I didn't know this service, awesome!

Reply:
Me neither before today, I found it linked from the GitHub docs (https://help.github.com/articles/ignoring-files/).